huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -1170,11 +1167,11 @@ func resourceComputeInstanceV2ImportState(d *schema.ResourceData, meta interface
 	return []*schema.ResourceData{d}, nil
 }
 
-// ServerV2StateRefreshFunc returns a resource.StateRefreshFunc that is used to watch
+// ServerV1StateRefreshFunc returns a resource.StateRefreshFunc that is used to watch
 // an HuaweiCloud instance.
```

Comment:
Lines 1170 and 1171 can be combined into one.

Reply:
fixed